package.json

```diff
@@ -30,6 +30,7 @@
     "@types/node": "20.9.2",
     "@types/react": "18.2.37",
     "@types/react-dom": "18.2.15",
+    "axios-mock-adapter": "^1.22.0",
```

Comment:
Do you need axios mock adapter?

Jest should be able to mock Axios.

Reply:
This is a long story :sweat_smile: 
I have tried sth along the lines of 
```
jest.mock("axios");

describe("when API call is successful", () => {
  it("should return users list", async () => {
    // given
    const response = 'sth else';
    axios.get.mockResolvedValueOnce(response);
```
but I kept getting undefined method error from axios.get

I also tried `jest-mock-axios` package and got the same issue.
So I turned to this package and it worked (i guess so)